testsuite/domain/src/test/java/org/jboss/as/test/integration/domain/slavereconnect/SlaveReconnectTestCase.java

```diff
@@ -148,6 +151,14 @@ private void testReconnect(ReconnectTestScenario[] scenarios) throws Exception {
                 slaveReconnected = checkSlaveReconnected(masterClient);
             } while (!slaveReconnected && System.currentTimeMillis() < end);
 
+            //Wait for master servers to come up
+            end = System.currentTimeMillis() + 20 * ADJUSTED_SECOND;
```

Comment:
20 sounds too low.

If the test is going to fail, waiting a long time for it to happen is no big deal. But a false failure because of a short timeout is bad.